kuma/core/helpers.py

```diff
@@ -8,6 +9,7 @@
 from django.utils.encoding import smart_str
 from django.utils.http import urlencode as urlencode_util
 from django.utils.tzinfo import LocalTimezone
+from django.templatetags.static import static
```

Comment:
Sorry I missed that earlier, that must be `from django.contrib.staticfiles.templatetags.staticfiles import static`